RUFAS/routines/field/manager/crop_schedule.py

```diff
@@ -4,13 +4,64 @@
 from RUFAS.routines.field.crop.harvest_operations import HarvestOperation, FINAL_HARVEST_OPERATIONS
 from RUFAS.routines.field.manager.events import PlantingEvent, HarvestEvent
 
-"""
-The `CropSchedule` module allows users to specify a pattern for planting and harvesting a certain crop that can be
-repeated over a specified number of years, with specified breaks in between repetitions of the pattern.
-"""
-
 
 class CropSchedule(Schedule):
+    """
+    A class for defining a schedule for planting and harvesting crops, allows users to specify a pattern for planting
+    and harvesting a certain crop that can be repeated over a specified number of years, with specified breaks in
+    between repetitions of the pattern.
+
+    Parameters
+    ----------
+    name : str
+        Reference to the name of this crop schedule that will be used to distinguish this schedule from others.
+    crop_reference : str
+        Reference to the name of the crop that will be used to identify the correct crop specifications.
+    planting_years : List[int]
+        Years in which the crop is planted.
+    planting_days : List[int]
+        Julian days on which the crop is planted.
+    harvest_years : List[int]
+        Years in which the crop is harvested.
+    harvest_days : List[int]
+        Julian days on which the crop is harvested.
+    harvest_operations : List[str]
+        Operations with which the crop is harvested.
+    use_heat_scheduling : bool, optional, default False
+        Indicates if heat scheduling should be used to determine when the crop is harvested.
+    planting_skip : int, optional, default 0.0
+        Number of years to skip between planting cycles.
+    harvesting_skip : int, optional, default 0.0
+        Number of years to skip between harvesting cycles.
+    pattern_repeat : int, optional, default 0.0
+        Number of times the specified crop planting and harvesting pattern should be repeated.
+
+    Attributes
+    ----------
+    crop_reference : str
+        Identifier for the crop associated with this schedule.
+    planting_years : List[int]
+        List of years for each planting event.
+    planting_days : List[int]
+        Corresponding Julian days for planting.
+    planting_skip : int
+        Number of years to skip between planting events.
+    harvest_years : List[int]
+        List of years for each harvesting event.
```

Comment:
```suggestion
        List of years in which planting events will occur.
    planting_days : List[int]
        Corresponding Julian days for planting.
    planting_skip : int
        Number of years to skip between planting events.
    harvest_years : List[int]
        List of years in which harvesting events will occur.
```
I think "List of years for each event" makes it sound like each event has a list of years, but one event can only be associated with one year.